pkg/csi/recover/recover.go

```diff
@@ -158,7 +170,8 @@ func (r FuseRecover) recover() {
 
 	for _, point := range brokenMounts {
 		glog.V(4).Infof("Get broken mount point: %v", point)
-		r.umountDuplicate(point)
+		// do not umountDuplicate because if app container restart, umount duplicate mount may lead to recover successed but can not access data
```

Comment:
You can add comments with the issue link.

Reply:
ok.